nativelib/src/main/scala/scala/scalanative/runtime/ieee754tostring/ryu/RyuRoundingMode.scala

```diff
@@ -35,34 +35,19 @@ package scala.scalanative
 package runtime
 package ieee754tostring.ryu
 
-object RyuRoundingMode extends Enumeration {
-
-  val CONSERVATIVE: RyuRoundingMode = new conservativeRM()
-
-  val ROUND_EVEN: RyuRoundingMode = new roundEvenRM()
-
-  abstract class RyuRoundingMode {
-
-    def acceptUpperBound(even: Boolean): Boolean
-
-    def acceptLowerBound(even: Boolean): Boolean
-
-  }
-
-  private[this] class conservativeRM extends RyuRoundingMode {
+abstract class RyuRoundingMode {
+  def acceptUpperBound(even: Boolean): Boolean
+  def acceptLowerBound(even: Boolean): Boolean
+}
 
+object RyuRoundingMode {
+  object Conservative extends RyuRoundingMode {
```

Comment:
By making the instances of `RoundingMode` be `object`-s, Interflow now specializes the floatToString/doubleToString to the rounding mode. This means two (up to) copies of the method where rounding mode is statically known and moth of the `acceptUpperBound` and `acceptLowerBound` are constant and folded away. In practice we only use conservative mode now, so it's exactly one copy.